src/container/BooksList.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from '../components/Book';
+
+class BooksList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.props = props;
+  }
+
+  render() {
+    const { books } = this.props;
+    return (
+      <div>
+        <table>
+          <thead>
+            <tr>
+              <th>ID</th>
+              <th>Title</th>
+              <th>Category</th>
+            </tr>
+          </thead>
+          <tbody>
+            {books.map(e => (
+              <Book id={e.id} title={e.title} category={e.category} key={e.id} />
+            ))}
```

Comment:
You forgot the task below 
![Screen Shot 2020-01-24 at 10 41 26 AM](https://user-images.githubusercontent.com/40712459/73059543-38f7c580-3e96-11ea-8400-4ad6626d1835.png)

which is also linked to this 
![Screen Shot 2020-01-24 at 10 46 59 AM](https://user-images.githubusercontent.com/40712459/73059867-ee2a7d80-3e96-11ea-96f6-a92e0037bf67.png)


Basically you have to  DISPATCH `removeBook` action  and pass it  into BOOK component here as a prop (read about `mapDispatchToProps`)